### ISSUE TITLE
Automatic categorization causes error on startup

### ISSUE THREAD

**Issue by za267** [minetest-mods/unified_inventory#239]
I get a crash if I use the automatic categorization feature.  I've disabled it temporarily in minetest.conf, but the error was:

```
ERROR[Main]: ModError: Runtime error from mod 'unified_inventory' in callback on_mods_loaded(): .../.minetest/mods/unified_inventory/default-categories.lua:47: attempt to index a nil value
```

**Comment by SmallJoker** [comment]
You enabled a mod that contains an invalid ore registration (`minetest.register_ore`) of type `"scatter"` . The field `ore` does not refer to a known node. After disabling unified_inventory, you should see an error similar to `ERROR[Main]: NodeResolver: failed to resolve node name 'mymod:my_invalid_node'.`.

Fixed in e7d0362

**Comment by za267** [comment]
Thanks!
